src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/menu_thumbnail_images.html.twig

```diff
@@ -24,14 +24,14 @@
  *#}
 
 {% block category_menu_thumbnails %}
-  {% if menuThumbnailImages is defined and menuThumbnailImages is not empty %}
+  {% if preview_images is defined and preview_images is not empty %}
     <div>
-      {% for menuThumbnail in menuThumbnailImages %}
+      {% for menuThumbnail in preview_images %}
```

Comment:
I wonder if there would be a way to simplify these three form types into a single one, although I'm not sure it's possible but here are my suggestions:
- menu_thumbnail_images and cover_image seem to be very similar except for the fact the the first one has a list of images, but if you had a component relying always on a list it could also work for cover_image only it would always have an array of one element
- cover_image and category_thumbnail_image also seem quite similar, mostly the difference id that cover_image handles a delete method, but if you had a `can_be_deleted` option, then the display of the delete buttons could be conditioned

So I was thinking maybe (bu you'll know better than me) we could have a single form type that accepts a list of images and can option out the delete button in which case one form type and one template would be enough. But if it gets too complex it may become counterproductive, so let me know what you think about it

Reply:
Added those 3 together into 1